lib/awful/client.lua

```diff
@@ -59,6 +59,56 @@ client.property = {}
 client.shape = require("awful.client.shape")
 client.focus = require("awful.client.focus")
 
+--- The POSIX signal table.
+--
+-- This table was extracted from
+--
+--    man 7 signal
```

Comment:
Sorry, but this is non-portable. POSIX doesn't define the numbers for the signals, this is Linux-specific.

http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/signal.h.html